src/Model/Filter/Base.php

```diff
@@ -72,12 +72,18 @@ public function __construct(string $name, Manager $manager, array $config = [])
             'flatten' => true,
             'beforeProcess' => null,
         ];
-        $config += $defaults;
+
+        if (isset($config['field'])) {
+            throw new \InvalidArgumentException(
+                'The `field` option has been renamed to `fields`.'
+            );
+        }
+        $config['fields'] = (array)$config['fields'];
         $this->setConfig($config);
 
         if (
-            (empty($config['field']) && $config['field'] !== '0') ||
```

Comment:
Is this safe to change? As the check is specifically around the falsey values in php.